webrtc.html

```diff
@@ -4855,51 +4855,46 @@ <h2>Methods</h2>
             <dt><code>getSenders</code></dt>
             <dd>
               <p>Returns a sequence of <code><a>RTCRtpSender</a></code> objects
-              representing the RTP senders that are currently attached to this
-              <code><a>RTCPeerConnection</a></code> object.</p>
-              <p>The <dfn id=
+              representing the RTP senders that belong to non-stopped
+              <code><a>RTCRtpTransceiver</a></code> objects currently attached
+              to this <code><a>RTCPeerConnection</a></code> object.</p>
+              <p>When the <dfn id=
               "dom-peerconnection-getsenders"><code>getSenders</code></dfn>
-              method MUST return the result of executing the
-              <code><a>CollectSenders</a></code> algorithm.</p>
-              <p></p>
-              <p>We define the <dfn>CollectSenders</dfn> algorithm as
```

Comment:
The CollectSenders algorithm is still referenced from addTrack, now it is no longer defined. Update that step to get the result of getSenders()?